cmd/krew/cmd/index.go

```diff
@@ -54,9 +55,31 @@ each configured index in table format.`,
 	},
 }
 
+var indexAddCmd = &cobra.Command{
+	Use:   "add",
+	Short: "Add a new index",
+	Long: `Configure a new index to install plugins from.
+
+Example:
+  kubectl krew index add index-name index-url`,
```

Comment:
This isnt really an example, but a placeholder. 

Lets use default index as example